dozer-types/protos/cloud.proto

```diff
@@ -66,23 +65,24 @@ message AppHealthRequest {
 message AppHealthResponse {
   bool healthy = 1;
 }
-message DeploymentHealthCheckRequest {
+message DeploymentStatusRequest {
   string app_id = 1;
+  // if not specified, will get latest version
+  optional uint32 version = 2;
 }
-message DeploymentHealthCheckResponse {
-  bool all_healthy = 1;
-  repeated DeploymentHealth k8_deployments = 2;
-  uint32 latest_deployment = 3;
-  repeated string errors = 4;
+message DeploymentStatusResponse {
+  repeated DeploymentHealth k8_deployments = 1;
```

Comment:
Why it's `repeated` while the request has a specified version?

Reply:
it's the `Deployments` in k8s - to show sth like this
<img width="1465" alt="Screenshot 2023-09-12 at 21 40 11" src="https://github.com/getdozer/dozer/assets/17896317/793246d0-116d-46d0-9bc3-63b4bb72af66">

Reply:
Got it